components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/test/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/util/DCRMUtilsTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.dcr.endpoint.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.oauth.dcr.exception.DCRMException;
+import org.wso2.carbon.identity.oauth2.dcr.endpoint.Exceptions.DCRMEndpointException;
+import org.wso2.carbon.identity.oauth2.dcr.endpoint.dto.RegistrationRequestDTO;
+import org.wso2.carbon.identity.oauth2.dcr.endpoint.dto.UpdateRequestDTO;
+
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DCRMUtilsTest {
+    private List<String> redirectUris = new ArrayList<>();
+    private List<String> grantTypes = new ArrayList<>();
+    private final String client_name = "Application";
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        redirectUris.add("https://op.certification.openid.net:60845/authz_cb");
+        grantTypes.add("authorization_code");
+    }
+
+    @Test
+    public void testGetApplicationRegistrationRequest() throws Exception {
+        RegistrationRequestDTO  registrationRequestDTO = new RegistrationRequestDTO();
+        registrationRequestDTO.setClientName(client_name);
+        registrationRequestDTO.setRedirectUris(redirectUris);
+        registrationRequestDTO.setGrantTypes(grantTypes);
+        Assert.assertNotNull(DCRMUtils.getApplicationRegistrationRequest(registrationRequestDTO));
+        Assert.assertEquals(DCRMUtils.getApplicationRegistrationRequest(registrationRequestDTO).getClientName(), client_name);
```

Comment:
Fix formatting. Break this line if it exceeds 120 characters. 
Format all the classes.